### PR TITLE
Bug fix club comment

### PR DESCRIPTION
**Pull request by LudoBike**
Bug fix cf issue #100

### CODE REVIEW

**Review by sachabinder — APPROVED** [review]
It works perfectly, good job bro :100:

**Review by sachabinder — DISMISSED** [review]

**Review by sachabinder — APPROVED** [review]

File: react/src/components/commentForm.jsx [hunk 1]
```diff
@@ -77,27 +82,20 @@ class CommentForm extends React.Component {
...
                     {options}
                 </select>
-                <button className="button green-button news-card-button" type="submit"><i className="far fa-comment"></i></button>
             </div>
 
             return (
-            <form method="post" onSubmit={this.handleSubmit.bind(this)}>
-                {field1}
+            <form className="news-card-edit-comment-container" method="post" onSubmit={this.handleSubmit.bind(this)}>
                 Publier en tant que :
```
Comment (LudoBike) [inline react/src/components/commentForm.jsx:89]: I think "Commenter en tant que :" would be better

**Comment by LudoBike** [comment]
And perhaps it would be better if the scroll list to select clubs is left align instead of centered

**Review by sachabinder — APPROVED** [review]

**Comment by LudoBike** [comment]
Looks great, you can merge imo

**Pull request merged**